### ISSUE TITLE
Support batchSize>1

### ISSUE THREAD

**Issue by bigaidream** [nicholas-leonard/drmad#12]
Currently, [drmad_mnist](https://github.com/nicholas-leonard/drmad/blob/master/drmad_mnist.lua) does not support `batchSize > 1`.

**Comment by bigaidream** [comment]
No need at this moment. Focusing on the new API for learning rates, weight decay and momentum.

**Comment by nicholas-leonard** [comment]
Will do.

Nicholas Léonard
917-741-7570
